wagtail/admin/views/pages/edit.py

```diff
@@ -32,6 +34,81 @@
 )
 
 
+class BaseSidePanel(Component):
+    def __init__(self, page):
+        self.page = page
+
+    class Toggle(Component):
+        def __init__(self, panel):
+            self.panel = panel
+
+        aria_label = gettext_lazy("Toggle")
+        template_name = "wagtailadmin/pages/editor_sidebar_panels/toggle.html"
+        icon_name = ""
+
+        def get_context_data(self, parent_context):
+            return {
+                "toggle": self,
+                "panel": self.panel,
+                "page": self.panel.page,
+            }
+
+    @property
+    def toggle(self):
+        return self.Toggle(self)
+
+    def get_context_data(self, parent_context):
+        return {
+            "panel": self,
+            "page": self.page,
+        }
+
+
+class StatusSidePanel(BaseSidePanel):
+    name = "status"
+    title = gettext_lazy("Status")
+    template_name = "wagtailadmin/pages/editor_sidebar_panels/status.html"
+    order = 100
+
+    class Toggle(BaseSidePanel.Toggle):
+        aria_label = gettext_lazy("Toggle status")
+        icon_name = "site"  # TODO Find the real icon
+
+
+class HistorySidePanel(BaseSidePanel):
+    name = "history"
+    title = gettext_lazy("History")
+    template_name = "wagtailadmin/pages/editor_sidebar_panels/history.html"
+    order = 200
+
+    class Toggle(BaseSidePanel.Toggle):
+        aria_label = gettext_lazy("Toggle history")
+        icon_name = "history"
```

Comment:
How are we going to reconcile this with the fact that for now we want History to be a link to the existing History page? Do we need to create a new `toggle.html` just for this, and move the existing link in `page_edit_header.html` to that template?

Reply:
Oh, I didn't know this. I think it should be a separate thing. I'll remove the history panel for now